### PR TITLE
feat(apm) Add a rough draft of related events

### PR DESCRIPTION
**Pull request by markstory**
Use the current event's traceid to find other related events. The display is rough but it is a start.

Screen Shot 2019-07-30 at 3 58 18 PM

Refs SEN-860

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    return (
+      <Container>
+        <Title>
+          <InlineSvg src="icon-link" size="12px" /> {t('Related Events')}
+        </Title>
+        {events.length <= 1 ? (
+          <Card>{t('No related events found.')}</Card>
+        ) : (
+          events
+            // Don't show the current event. We don't have negation conditions
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx]: Can't we query for `!id:<id>`?

File: src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            // in our API so we have to do this here.
+            .filter(item => item.id !== event.id)
+            .map(item => {
+              const eventUrl = {
+                pathname: location.pathname,
+                query: {
+                  ...omit(location.query, MODAL_QUERY_KEYS),
+                  eventSlug: `${item['project.name']}:${item.id}`,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx]: Is this something generic that we would want in a util function?

File: src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                },
+              };
+              const project = {slug: item['project.name']};
+              return (
+                <Card key={item.id}>
+                  <EventLink to={eventUrl} data-test-id="linked-event">
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx]: Is `data-test-id` needed?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx]: I'll give that a shot.

File: src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx (hunk 3 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx]: I'm going to use it in an acceptance test.

**Review by billyvg — APPROVED** [review]

**Pull request merged**